shell/platform/darwin/common/framework/Source/FlutterBinaryMessengerRelay.mm

```diff
@@ -14,7 +14,7 @@ @implementation FlutterBinaryMessengerRelay
 - (instancetype)initWithParent:(NSObject<FlutterBinaryMessenger>*)parent {
   self = [super init];
   if (self != nil) {
-    self.parent = parent;
+    _parent = parent;
```

Comment:
This is unrelated, but as discussed in Discord I accidentally duplicated the recent work to adopt FlutterBinaryMessengerRelay on macOS as part of debugging my delloc test not working, and this was the only substantive diff between our versions. `init` methods should not call `self` (which we violate a lot unfortunately, but this one was trivial to fix), so I had done a drive-by fix to this as part of moving it.

I can revert if you'd rather.

Reply:
Ah good call. Thanks.